tests/commands/install_uninstall_test.py

```diff
@@ -726,6 +726,51 @@ def test_commit_msg_legacy(commit_msg_repo, tempdir_factory, store):
     assert second_line.startswith('Must have "Signed off by:"...')
 
 
+def test_post_commit_integration(tempdir_factory, store):
+    path = git_dir(tempdir_factory)
+    config = [
+        {
+            'repo': 'local',
+            'hooks': [{
+                'id': 'post-commit',
+                'name': 'Post commit',
+                'entry': 'bash -c "touch post-commit.tmp"',
```

Comment:
no need for a subshell, it can just `touch post-commit.tmp` -- a little bit strange of an example though 🤔

Reply:
In my use-case I will create a "summary" of sorts for a commit and save it to disk. It was the first case that came to mind and should broaden the coverage the test suite.